src/Control/IPUtils.php

```diff
@@ -7,7 +7,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-namespace SilverStripe\Control\Util;
```

Comment:
After a 🛰 spacewalk return to your place!

Reply:
We'd have to target `master` with this change.

Reply:
okay, please merge #9039